src/python/grapl-template-generator/grapl_template_generator/grapl-templates/rust-grpc-service/{{cookiecutter.project_slug}}/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+/*
+This module just re-exports the server, client, and protos. No need to modify.
+*/
+
+pub mod server;
+pub mod client;
+
+pub mod {{cookiecutter.snake_project_name}} {
+    // In the future, this will be in rust-proto.
+    tonic::include_proto!("{{cookiecutter.snake_project_name}}");
+
+    pub fn get_socket_addr() -> String {
```

Comment:
This should probably return a `std::net::SocketAddr` instead of a String.

Reply:
That was actually the original plan (hence the name of the function) - but there's actually some places like
```
async fn _until_health() -> Result<(), Box<dyn std::error::Error>> {
    let endpoint = format!("http://{}", get_socket_addr());
    let channel = Channel::from_shared(endpoint)?.connect().await?;
```
that can't take a SocketAddr